sbnanaobj/StandardRecord/SRSlice.cxx

```diff
@@ -6,18 +6,9 @@
 
 #include "sbnanaobj/StandardRecord/SRSlice.h"
 
-#include <limits>
-#include <climits>
-
 namespace caf
 {
-  SRSlice::SRSlice():
-    producer(UINT_MAX),
-    charge(std::numeric_limits<float>::signaling_NaN()),
-    is_clear_cosmic(false),
-    nu_pdg(INT_MIN),
-    nu_score(std::numeric_limits<float>::signaling_NaN()),
-    self(INT_MIN)
+  SRSlice::SRSlice()
```

Comment:
Same comment as for the other class: an empty default constructor impedes some language features, and unless that is intended (in which case it would be good to add a comment stating it), the empty constructor is better gone.
```suggestion
```